README.md

```diff
@@ -1,25 +1,75 @@
 # Ember Resolver [![Build Status](https://travis-ci.org/ember-cli/ember-resolver.svg?branch=master)](https://travis-ci.org/ember-cli/ember-resolver)
 
 
-This project is tracking a new resolver based on ES6 semantics that has been extracted from (and used by) the following projects:
+This project provides the Ember resolver used by the following projects:
 
 * [ember-cli](https://github.com/ember-cli/ember-cli)
 * [ember-app-kit](https://github.com/stefanpenner/ember-app-kit)
 * [ember-appkit-rails](https://github.com/DavyJonesLocker/ember-appkit-rails)
 
 ## Installation
 
-Ember-resolver was previously a bower package, but since v1.0.1, it has become an ember-cli addon, and should be installed with `ember install`:
+Ember-resolver is an ember-cli addon, and should be installed with `ember install`:
+
 ```
 ember install ember-resolver
 ```
 
-If you're currently using ember-resolver v0.1.x in your project, you should uninstall it:
+## Feature flags
+
+This resolver package supports feature flags for experimental changes. Feature
+flag settings change how `ember-resolver` compiles into an ember-cli's
+`vendor.js`, so you should think of them as an application build-time option.
+
+Feature flags are set in an application's `config/environment.js`:
+
+```js
+/* eslint-env node */
+
+module.exports = function(environment) {
+  var ENV = {
+    emberResolver: {
+      features: {
+        EMBER_RESOLVER_MODULE_UNIFICATION: true
+      }
+    },
+    /* ... */
+```
+
+Note that you must restart your ember-cli server for changes to the `flags` to
```

Comment:
When we move to `ember-cli-build.js` this becomes more obvious, since no changes to ember-cli-build.js are detected and updated without a restart today.